drivers/minio/README.md

```
@@ -462,7 +474,7 @@ small correctness/lifecycle checks.
 
 | Flag | Default | Purpose |
 |---|---|---|
-| `-versioning-enabled true|false` | `false` | Enable versioning on the bucket. Reads will include version IDs when listing. |
+| `-versioning-enabled true|false` | `false` | With writers, enable versioning on the bucket; with readers only, discover and address existing versions without mutating bucket configuration. |
```

markdownlint-cli2 (0.23.2)

[warning] Table column count
Expected: 3; Actual: 4; Too many cells, extra data will be missing

(MD056, table-column-count)